src/components/PendingIndicator/PendingIndicator.tsx

```diff
@@ -14,20 +14,16 @@ export const IconSize = {
     extraSm: 20,
 }
 
-export const PendingIndicator: React.FC<PendingProps> = ({src, alt, size}) => {    
+export const PendingIndicator: React.FC<PendingProps> = ({ alt, size}) => {    
     return (
-        <Button>
+        <PendingImgWrapper>
             <PendingImg 
                 style={{maxWidth: `${size}px`}}
-                src={src} 
+                src={vamalakaLoading} 
                 alt={alt}
```

Comment:
so far alt is always the same so create default value for it ({ alt, size='Pending indicator'}). This way you dont have to repeat it on every usage